docs/index.md

```diff
@@ -9,7 +9,7 @@ more.
 
 [Installation](docs/getting-started/)
 
-## Example usage
+**Example usage**
```

Comment:
And it doesn't need to be imo, the logo links to it and having "Home" in the nav is ugly imo